mapbox/app/src/main/java/com/mapbox/services/android/testapp/directions/DirectionsV5Activity.java

```diff
@@ -132,6 +133,7 @@ public void accept(DirectionsResponse response) throws Exception {
     client.enqueueCall(new Callback<DirectionsResponse>() {
       @Override
       public void onResponse(Call<DirectionsResponse> call, Response<DirectionsResponse> response) {
+        Log.d(LOG_TAG, call.request().url().toString());
```

Comment:
If you meant to leave this line, let's add a more detailed message of what the developer is seeing.